crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -4074,7 +4078,24 @@ spec:
         type: object
     served: true
     storage: false
-  - name: v1alpha2
+  - additionalPrinterColumns:
+    - JSONPath: .status.workspaceId
+      description: The workspace's unique id
+      name: Workspace ID
+      type: string
+    - JSONPath: .status.phase
+      description: The current workspace startup phase
+      name: Phase
+      type: string
+    - JSONPath: .status.statusInfo
```

Comment:
I was confused why we have statusInfo and message at the same time, but PR check explained it in clear manner https://github.com/devfile/api/pull/221/checks?check_run_id=1567151866
CRD must be updated. Now it's fixed.